hash.c

```diff
@@ -3754,14 +3753,14 @@ inspect_hash(VALUE hash, VALUE dummy, int recur)
 }
 
 /*
- * call-seq:
- *   hsh.to_s     -> string
- *   hsh.inspect  -> string
+ *  call-seq:
+ *    hash.to_s -> new_string
  *
- * Return the contents of this hash as a string.
+ *  Returns a new \String containing the hash entries:
+ *    h = {foo: 0, bar: 1, baz: 2}
```

Comment:
I would just like to add, I agree with removing internal whitespace, I think it unecessary... :+1: